config/sync/field.field.paragraph.os2loop_section_page_info_block.os2loop_section_page_info_text.yml

```diff
@@ -0,0 +1,68 @@
+uuid: 5230c7c3-fccc-43b0-b44b-36515d34183e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.os2loop_section_page_info_text
+    - paragraphs.paragraphs_type.os2loop_section_page_info_block
+  module:
+    - better_formats
+    - text
+third_party_settings:
+  better_formats:
+    allowed_formats_toggle: 1
+    allowed_formats:
+      os2loop_section_page: os2loop_section_page
+      os2loop_question_answer_plain_text: 0
+      os2loop_question_answer_rich_text: 0
+      os2loop_post_comment_plain_text: 0
+      os2loop_post_comment_rich_text: 0
+      os2loop_documents_body: 0
+      os2loop_documents_rich_text: 0
+      os2loop_external: 0
+      os2loop_message_message: 0
+      os2loop_page: 0
+      os2loop_post: 0
+      os2loop_question_plain_text: 0
+      os2loop_question_rich_text: 0
+    default_order_toggle: 0
+    default_order_wrapper:
+      formats:
+        os2loop_question_answer_plain_text:
+          weight: '0'
+        os2loop_question_answer_rich_text:
+          weight: '0'
+        os2loop_post_comment_plain_text:
+          weight: '0'
+        os2loop_post_comment_rich_text:
+          weight: '0'
+        os2loop_documents_body:
+          weight: '0'
+        os2loop_documents_rich_text:
+          weight: '0'
+        os2loop_external:
+          weight: '0'
+        os2loop_message_message:
+          weight: '0'
+        os2loop_page:
+          weight: '0'
+        os2loop_post:
+          weight: '0'
+        os2loop_question_plain_text:
+          weight: '0'
+        os2loop_question_rich_text:
+          weight: '0'
+        os2loop_section_page:
+          weight: '0'
+id: paragraph.os2loop_section_page_info_block.os2loop_section_page_info_text
+field_name: os2loop_section_page_info_text
+entity_type: paragraph
+bundle: os2loop_section_page_info_block
+label: 'Information block'
+description: ''
+required: false
```

Comment:
Should the text be required? I vote “Yes!”

Reply:
Why not just let the user decide if they want to add a h2 or not above the text?
Not all content needs a header. There will often be a h1 right above it.
I vote "no!"